packages/suite/src/components/suite/Labeling/components/Metadata/index.tsx

```diff
@@ -40,9 +40,9 @@ const LabelButton = styled(Button)`
     text-align: left;
 `;
 
-const ActionButton = styled(Button)`
+const ActionButton = styled(Button)<{ isVisible?: boolean }>`
```

Comment:
`isVisible` seems to be used all the time below, no need to make it optional.

Reply:
initially there were 3 occurences of this component (you missed one of them). But while I was thinking about it I found out that I may completely [remove a block of code](https://github.com/trezor/trezor-suite/pull/2596/commits/892b94c86110f19e1067830e4a2e55e4951fb354) as the concern it was meant to solve is actually solved already.

Reply:
So in the end I kept the aformentioned block there as removing it caused the tests to fail. And it probably makes sense as it contains different copy.